compiler-plugin/src/test/scala/scalapb/compiler/ProtoValidationSpec.scala

```diff
@@ -305,6 +305,23 @@ class ProtoValidationSpec extends AnyFlatSpec with Matchers with ProtocInvocatio
     )
   }
 
```

Comment:
Please add another test for sealed_oneofs_extends (looks like there's just one for companion)

Reply:
nope, test for `sealed_oneof_extends` right above this one, on line 291

https://github.com/scalapb/ScalaPB/pull/1371/files/68a353f9de969743c9291b1b510065e3e52f7ae2#diff-a5d18260251210f14244e7f5ef58a497bca2972a74e5e4211df929ffde0b5ec4L291-L306